pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+# Fixes # (issue)
+## Description
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change, and any blockers that make your change a WIP*
+
+# Checklist:
+- [ ] I have manually tested my changes on desktop and mobile
+- [ ] The test suite passes locally with my changes
+- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
+- [ ] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
+- [ ] I have made any necessary changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] My PR is labeled [WIP] if it is in progress
```

Comment:
can WIP be in the title? or literally adding the label wip - just want to make sure this is clear. can everyone add labels at this level?

Reply:
Oh yeah! I was imagining this being in the title. I can make that more clear since some folks can't add lables